src/extension/gogoduck/doc/wps-acorn.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
In the three given example requests, the only difference appears to be the layer name and the TIME range.  Is it *really* necessary then to have the three examples, or would one be enough?  
Otherwise, it's more doc to keep up to date, more information for the reader to process.